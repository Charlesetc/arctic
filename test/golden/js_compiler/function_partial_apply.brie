define main [
  define f : x y z [ x ]
  define y [ 2 ]
  define f (f 2)
  f y "hi"
  f "hi" y
]
