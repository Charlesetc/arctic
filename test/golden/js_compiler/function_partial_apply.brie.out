const global_functions = {1: {},2: {"Integer,Function,String": fn_2_Integer__Function__String,"Integer,String,Function": fn_2_Integer__String__Function},3: {}};
function fn_2_Integer__Function__String(x,y,z) {
	return x;
}
function fn_2_Integer__String__Function(x,y,z) {
	return x;
}
function main() {
	var f = new_closure(2);
	var y = new_closure(3);
	var f = f.partial(2, "Integer");
	f.partial(y, "Function").partial("hi", "String").call();
	return f.partial("hi", "String").partial(y, "Function").call();
}
var test/golden/js_compiler/function_partial_apply.brie_main = new_closure(1)
console.log(main());